CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultActivityChains.uc

```diff
@@ -392,9 +394,47 @@ static function X2DataTemplate CreateSupplyInterceptionTemplate()
 	
 	Template.Stages.AddItem('Activity_SupplyRescue');
 
+	Template.PostStageSetup = AttachResCon;
+
 	return Template;
 }
 
+static function AttachResCon(XComGameState NewGameState, XComGameState_Activity ActivityState)
+{
+	local XComGameState_ResourceContainer ResConState;
+	local XComGameState_MissionSite MissionState;
+	local XComGameState_Reward MissionReward;
+	local XComGameStateHistory History;
+	local int x, y;
+	
+	History = `XCOMHISTORY;
+
+	MissionState = XComGameState_MissionSite(History.GetGameStateForObjectID(ActivityState.PrimaryObjectRef.ObjectID));
```

Comment:
In theory, you should check that it's the correct stage. However, it doesn't really matter in this case since the chain only has 1 stage anyway. Still, I think a comment that explains this fact would be useful